### PR TITLE
GEOMESA-1967 Add docs for geodocker-geomesa on local machine.

### PR DESCRIPTION
**Pull request by aheyne**
Signed-off-by: Austin Heyne <aheyne@ccri.com>

### CODE REVIEW

File: docs/tutorials/geodocker-geomesa/index.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
+GeoDocker GeoMesa
+===================
+
+The GeoDocker-GeoMesa project makes standing up a GeoMesa cluster
+running on Accumulo very easy. Currently we support running a
+GeoDocker-GeoMesa cluster locally, on a single machine, and on an
```
Comment (elahrvivaz) [inline docs/tutorials/geodocker-geomesa/index.rst]: slightly ambiguous - could we change to parens maybe? 'locally (on a single machine), or on an AWS cluster'

File: docs/tutorials/geodocker-geomesa/index.rst [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+AWS cluster. The tutorials below will guide you through the respective
```
Comment (elahrvivaz) [inline docs/tutorials/geodocker-geomesa/index.rst]: Lets add another sentence after 'AWS cluster' something like: 'Other deployment setups are possible, but we don't provide ready-made scripts.'

File: docs/tutorials/geodocker-geomesa/index.rst (hunk 1 above)
Comment (aheyne) [inline docs/tutorials/geodocker-geomesa/index.rst]: Changed

File: docs/tutorials/geodocker-geomesa/index.rst (hunk 2 above)
Comment (aheyne) [inline docs/tutorials/geodocker-geomesa/index.rst]: Changed

**Pull request merged**